pkg/fakes/log.go

```diff
@@ -1,4 +1,4 @@
-package test
```

Comment:
renaming the package to 'fakes' so that new contributors looking for integration tests don't get confused